packages/ragbits-core/src/ragbits/core/llms/base.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, model_name: str, default_options: Optional[LLMOptions] = None
             TypeError: If the subclass is missing the '_options_cls' attribute.
         """
         self.model_name = model_name
+        self.has_vision = False
```

Comment:
I don't think this is necessary - each LLM client implementation can decide whether (for any reason) it cannot handle a given request and react accordingly in this `call` method implementation (similarly how they handle other problems there already).

If we ever want `LLM` objects to report their capabilities, we should do it in a less ad-hoc manner, so it's not vision-specific. But I believe that can wait.

Reply:
already outdated - not used anymore